README.md

```diff
@@ -96,23 +96,32 @@ extension for your editor:
 
 ### Markdownlint
 
-Add a file named `.markdownlint.json` to the root of your project with the
+Add a file named `.markdownlint-cli2.jsonc` to the root of your project with the
 following content:
 
 ```json
 {
-   "extends": "./node_modules/@silvermine/standardization/.markdownlint.json"
+   "config": {
+      "extends": "./node_modules/@silvermine/standardization/.markdownlint.json"
+   },
+   "globs": [ "**/*.md" ],
+   "ignores": [
+      "node_modules",
+      "CHANGELOG.md"
+   ]
```

Comment:
Part of the reason for moving to markdownlint-cli2 is to share the globs, ignores, and eventually the rule config. This means we need to have some common  `.markdownlint-cli2.*js` file other projects extend.

Would it work to have something like `.markdownlint-cli2.shared.cjs` (or some other name that makes sense) that this project, the tests, and other projects extend? Possibly something like:

```diff
diff --git a/.markdownlint-cli2.cjs b/.markdownlint-cli2.cjs
new file mode 100644
index 0000000..0e4e517
--- /dev/null
+++ b/.markdownlint-cli2.cjs
@@ -0,0 +1,14 @@
+'use strict';
+
+const sharedStandards = require('./.markdownlint-cli2.shared.cjs');
+
+module.exports = {
+
+   ...sharedStandards,
+
+   ignores: [
+      ...sharedStandards.ignores,
+      'tests/markdownlint/**/*.md',
+   ],
+
+};
diff --git a/.markdownlint-cli2.shared.cjs b/.markdownlint-cli2.shared.cjs
new file mode 100644
index 0000000..8c2098e
--- /dev/null
+++ b/.markdownlint-cli2.shared.cjs
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+
+   config: require('./.markdownlint.json'),
+
+   globs: [ '**/*.md' ],
+
+   ignores: [
+      'node_modules',
+      'CHANGELOG.md',
+   ],
+
+};
diff --git a/tests/markdownlint/.markdownlint-cli2.cjs b/tests/markdownlint/.markdownlint-cli2.cjs
new file mode 100644
index 0000000..5d322e1
--- /dev/null
+++ b/tests/markdownlint/.markdownlint-cli2.cjs
@@ -0,0 +1,13 @@
+'use strict';
+
+const sharedStandards = require('../../.markdownlint-cli2.shared.cjs');
+
+module.exports = {
+
+   ...sharedStandards,
+
+   // Disable the default globs since ./bin/run-markdownlint-tests.sh supplies the files
+   // one at a time.
+   globs: [],
+
+};
```